app/src/main/java/com/mapbox/navigation/examples/basics/ShowBuildingExtrusionsActivity.kt

```diff
@@ -244,19 +231,13 @@ class ShowBuildingExtrusionsActivity : AppCompatActivity() {
      * on the map.
      */
     private val routesObserver: RoutesObserver = RoutesObserver { routes ->
-        // RouteLine: wrap the DirectionRoute objects and pass them
-        // to the MapboxRouteLineApi to generate the data necessary to draw the route(s)
-        // on the map.
-        val routeLines = routes.map { RouteLine(it, null) }
-
-        routeLineApi.setRoutes(
-            routeLines
-        ) { value ->
-            // RouteLine: The MapboxRouteLineView expects a non-null reference to the map style.
-            // the data generated by the call to the MapboxRouteLineApi above must be rendered
-            // by the MapboxRouteLineView in order to visualize the changes on the map.
-            mapboxMap.getStyle()?.apply {
-                routeLineView.renderRouteDrawData(this, value)
+        CoroutineScope(Dispatchers.Main).launch {
```

Comment:
I'd rather use callbacks in the examples and let developers figure out the preferred threading however they like. Callbacks are understandable to everyone, while coroutines might be a steeper learning curve.

Reply:
Reverted to what we had earlier. /cc @LukasPaczos